maquetado/styles.css

```diff
@@ -18,10 +19,60 @@ body {
   background: linear-gradient(to bottom, transparent, #f5c85f, transparent);
 }
 
+.logo {
+  max-width: 308px;
+  width: 85%;
+}
+
+.features-section {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: baseline;
+  justify-content: space-around;
+  align-content: center;
+  padding: 50px 15px;
+  margin-top: 50px;
+  background-color: #EEEEEE;
```

Comment:
colors that look like #AABBCC can be written like this: #ABC, in this case it would be #EEE